### PR TITLE
Migrate deployment from scripts to travis

### LINKED ISSUE #7: [Sling-framework] Use just .travis.yml to get rid of unnecessary logic on sh scripts
### Issue Description
Currently, the before_install and the after_success scripts are being used, but according to [this](https://docs.travis-ci.com/user/deployment/npm/) guide, it shouldn't be necessary.

### Expected Behaviour
The whole release process should be simpler and just use the .travis.yml file.

### PR DESCRIPTION
**Pull request by ayr-ton**
#### Short description of what this resolves:
Fix #7 

#### Changes proposed in this pull request:

- Remove scripts from .travis
- Use travis npm deployment functionality as described in https://docs.travis-ci.com/user/deployment/npm/

### CODE REVIEW

**Comment by joselitojunior1** [comment]
Discussion on #7

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "version-bump": "lerna publish --skip-git --skip-npm",
     "prepublish-packages": "npm install",
```
Comment (leofavre) [inline package.json]: This should be removed.

File: package.json [hunk 2]
```diff
@@ -8,6 +8,7 @@
     "version-bump": "lerna publish --skip-git --skip-npm",
```
Comment (leofavre) [inline package.json]: This should be removed.

File: package.json [hunk 3]
```diff
@@ -8,6 +8,7 @@
...
     "prepublish-packages": "npm install",
     "publish-packages": "lerna exec --bail=false -- npm publish",
```
Comment (leofavre) [inline package.json]: This should be removed.

File: package.json [hunk 4]
```diff
@@ -64,6 +65,8 @@
     "rollup": "^0.66.0",
     "rollup-plugin-commonjs": "^9.1.6",
     "rollup-plugin-node-resolve": "^3.4.0",
+    "semantic-release": "^15.9.17",
+    "semantic-release-monorepo": "^6.1.1",
     "semistandard": "^12.0.1",
```
Comment (leofavre) [inline package.json]: Some packages should be installed:

* @semantic-release/changelog
* @semantic-release/npm
* @semantic-release/git

**Comment by ayr-ton** [comment]
yay! \o/
Happy to help! 😄

**Pull request merged**